raiden/tests/integration/network/proxies/test_secret_registry.py

```diff
@@ -0,0 +1,212 @@
+from collections import defaultdict
```

Comment:
What's the logic of moving the contract proxy tests from `integration/contracts/ to `integration/network/proxies/` ? To match the directory structure of the client?

Reply:
Yes, otherwise it's unnecessarily hard to find the matching tests